docs/python/comparisons_and_logic.md

```diff
@@ -139,8 +139,10 @@ False
       ![Secure lock](https://preview.redd.it/j7i00nnpx6211.jpg?width=640&crop=smart&auto=webp&s=d4b58226076d45f0c637fd3789d3ccb547a4a54a){ width=50% }
     </figure>
 
-    It's your time to check if a password meets certain requirements. 
-    Use the following defined variables `password_length` and 
+    It's your time to check if a certain yet unknown (!) password meets 
+    the requirements. However, you won't need the password itself.
+
```

Comment:
Vielleicht sollten wir die Aufgabe etwas anders aufbauen. Es gibt vl zwei 'Kriterien' (min_password_length & special_character_allowed) und dann gibt es noch zwei Variablen über das echte passwort (password_length und has_special_character). und dann werden diese überprüft? Aber nur ein Vorschlag.

Reply:
How about:

## Evaluate password security requirements

You are given two variables that describe properties of a password:
* `password_length` - represents how many characters are in the password (integer)
* `has_special_characters` - represents whether the password contains special characters (True/False)

Variables to use:
```python
password_length = 18
has_special_characters = False
```

Task:
Write code that checks if this password is secure based on these requirements:
1. The password must be longer than 10 characters
2. The password must contain special characters

Use comparisons and logical operators to create a single expression that evaluates whether BOTH requirements are met.

Reply:
Nice! I think now it's clearer.

Reply:
Perfect, I'll rephrase the second part of the exercise as well! 😄